.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  semi: true,
+  trailingComma: "all",
+  printWidth: 120,
+  tabWidth: 2,
+  arrowParens: "avoid",
```

Comment:
oof

Reply:
what you want

Reply:
Doesn't really matter. I can lean either way on arrow parens. The prettier 2 rel notes had good reasons, but in the end it doesnt even matter